pkg/middleware/auth.go

```diff
@@ -271,9 +271,31 @@ func (a Auth) AuthorizeAPI(next http.Handler) http.Handler {
 				RenderError(errors.New("token expired"), http.StatusUnauthorized, w, r)
 				return
 			}
+
+			rawClaimsSdkKeys, ok := claims["sdk_keys"].([]interface{})
+			if !ok {
+				RenderError(errors.New("Invalid claims: sdk_keys not found, or have the wrong type"), http.StatusUnauthorized, w, r)
+				return
+			}
+			claimsSdkKeys := make([]string, len(rawClaimsSdkKeys))
```

Comment:
I don't think this slice is necessary since you can validate the requisite param while iterating over the `rawClaimsSdkKeys`. So a single loop is all you need.

Reply:
agree